### DISCUSSION TITLE
[Feature] Associate auto upload folders from mobile to album / tag

### DISCUSSION THREAD

**Discussion by xsben2** [immich-app/immich#11995] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When auto-download is enabled from mobile, we should be able to specify the album or (better) one or more tags (see #11730) per folder selected for auto-download.

This would be useful to know the photos that arrived "in bulk" from the mobile to classify them in the albums.
This can also be useful to know from which source (which phone (even if this info is in the exif tags) and especially the directory (whatapps, screenshot, camera ...)

![image](https://github.com/user-attachments/assets/56a332df-8db1-405d-af8f-102163311513)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1678

**Reply by xsben2** [reply]
> #1678

#11902 seems to be closer to my proposal, because it allows to choose the album on the server side.

All that's missing is the same thing with the tags :) #11730

**Comment by maxshcherbina** [comment]
While I appreciate the "Sync Albums" feature, I don't want all my albums to be uploaded in separate folders - given this, I would still like to see auto-tagging implemented as described here.

**Reply by xsben2** [reply]
@bo0tzz is it possible to reopen this discussion ?

It lacks the functionality with tags, and the ability to select a server-side album different from the mobile-side one.

Or should I start another discussion with what has already been added?

Sorry, if this is not the right method..
